src/style.css

```
@@ -1,3 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --main-accent-color: #00acdc;
+  --white: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
+  min-width: 1024px;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: block;
+  width: 100%;
+  height: 100%;
+  flex-shrink: 0;
+  align-items: center;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  margin-left: 20px;
+}
+
+.nav__link:first-child {
+  margin-left: 0;
+}
+
+.nav__item {
+  display: flex;
+  text-decoration: none;
+  color: var(--text-main-color);
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  height: 100%;
+  transition: color 0.3s;
+  line-height: 60px;
+}
+
+.nav__item:hover {
+  color: var(--main-accent-color);
+}
+
+.nav__item.is-active {
```

```suggestion
.is-active {
```
Do not use a combined selector without neccasserity